### PR TITLE
Rebased etcd

### PR DESCRIPTION
**Pull request by nikitamikhaylov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):

- Not for changelog (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
#10376

### CODE REVIEW

File: tests/config/config.d/zookeeper.xml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 <yandex>
     <zookeeper>
-        <implementation>testkeeper</implementation>
+        <implementation>etcdkeeper</implementation>
```
Comment (qoega) [inline tests/config/config.d/zookeeper.xml:3]: probably we need to change it from test environment to check different zookeeper alternatives in tests?

File: docker/test/stateless/run.sh [hunk 2]
```diff
@@ -3,6 +3,10 @@
 # fail on errors, verbose and export all env variables
 set -e -x -a
 
+service etcd start && sleep 5
+
+echo "Etcd"
```
Comment (qoega) [inline docker/test/stateless/run.sh:8]: better make it optional for case when we want to test etcd?

Comment (nikitamikhaylov) [reply docker/test/stateless/run.sh:8]: Of course, It is just for test here in PR.

**Comment by alexey-milovidov** [comment]
No need for Etcd, let's drop it :tada:

**Comment by simPod** [comment]
What is the alternative?

**Comment by alexey-milovidov** [comment]
@simPod https://github.com/ClickHouse/ClickHouse/issues/15090#issuecomment-810180249

@Maokaman1 The chance that this integration with Etcd started to work was neglible.

**Pull request closed without merging**